cabal.GHC-8.10.2.config

```diff
@@ -145,7 +145,6 @@ constraints: any.Cabal ==3.2.0.0,
              any.primitive ==0.7.0.1,
              any.process ==1.6.9.0,
              any.profunctors ==5.5.2,
-             any.random ==1.1,
```

Comment:
Although `cryptol`'s `.cabal` file no longer depends on `random` directly, `random` will still be a transitive dependency by way of `tf-random` (even though no `random` code paths are used in `cryptol` any more). I'd favor just leaving the `.config` files alone, since `random` will still be a part of the build plan.